src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowTests.cs

```diff
@@ -1824,252 +1824,187 @@ public static void M8()
 " + suffix;
 
             CreateCompilation(source).VerifyDiagnostics(
-                // (61,58): error CS0165: Use of unassigned local variable 'a'
+                // (87,58): error CS0165: Use of unassigned local variable 'a'
                 // /* NDA --> NDA */       { int a; if (x               ? F(a) : F(b)) b = c; else d = c; } // Error
```

Comment:
Are these inline comments all correct in the original code now?

Reply:
I believe so, but I'll make another pass to make sure. I actually suspect that they were originally written for a test in the native compiler and never updated to reflect the breaking change in Roslyn.

Reply:
By restoring native compiler behavior, a lot of the places that say OK in this test went from inaccurate to accurate again.￼

Reply:
Ok, so all the comments in here were pretty much just bad... got it.

Reply:
Just in case you were curious. There are 60 `// Error` comments in the test source, and now 60 calls to `Diagnostic` in the arguments to VerifyDiagnostics, and 60 `// Error` comments in the arguments to VerifyDiagnostics. So it looks like the comments are now accurate, FWIW.